conf/application.conf

```diff
@@ -230,7 +230,8 @@ urls {
     continueSchemeRegistration = "http://localhost:8200/register-pension-scheme/continue-registration"
     viewSchemeDetails = "http://localhost:8200/register-pension-scheme/scheme-details/%s"
     registerScheme = "http://localhost:8200/register-pension-scheme/what-you-will-need-before-start"
-    deleteSubscription = "http://localhost:8200/register-pension-scheme/discard"
+    deleteSubscription = "http://localhost:8200/register-pension-scheme/discard-scheme"
+    deleteSubscriptionRacdac = "http://localhost:8200/register-pension-scheme/discard-racdac"
```

Comment:
Have these urls been added to app-config-base?

Reply:
That's the PR that has been raised for this:
https://github.com/hmrc/app-config-base/pull/6004